### DISCUSSION TITLE
How must I build a Tor website?

### DISCUSSION THREAD

**Discussion by doctorstrange2001** [community/community#31950] [Other Feature Feedback, Questions, & Ideas]
Hi, I am 21 years old, I am from the Netherlands and today I created a account here, in order to build a website for Tor with Onionshare. I whatched many YouTube video's on how to do that, but I still do NOT know! Can anyone, especially if you're Dutch, help me with it?

My email is:
jvanderheide2001@gmail.com

**Comment by airtower-luna** [comment]
You're unlikely to get help if you just post a vague topic and then expect people to mail you. It's best if you ask your actual question(s), then people can reply if they have an answer.

Also note that when asking for help with a specific software it's generally best to ask in the relevant repository (in this case I assume that's https://github.com/onionshare/onionshare) or the support channels listed there. :wink:
